<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1400</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>441智能家居:SmartHomeX</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>source/logo.png</normaloff>source/logo.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>300</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="title">
           <string>环境信息</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="label_temperature">
             <property name="font">
              <font>
               <family>苹方-简</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>温度：NULL</string>
             </property>
             <property name="textFormat">
              <enum>Qt::AutoText</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_humidity">
             <property name="font">
              <font>
               <family>苹方-简</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>湿度：NULL</string>
             </property>
             <property name="textFormat">
              <enum>Qt::RichText</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_beam">
             <property name="font">
              <font>
               <family>苹方-简</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>光照：NULL</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>400</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="title">
           <string>识别信息</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QLabel" name="label_face_res">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
             <property name="text">
              <string>未知信息</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_name">
             <property name="text">
              <string>姓名：NULL</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_staus">
             <property name="text">
              <string>身份：NULL</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_time">
             <property name="text">
              <string>时间：NULL</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>苹方-简</family>
           </font>
          </property>
          <property name="text">
           <string>Made by 441 ©2023</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="minimumSize">
         <size>
          <width>900</width>
          <height>700</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>900</width>
          <height>800</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>摄像头</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QLabel" name="label_camera">
           <property name="minimumSize">
            <size>
             <width>900</width>
             <height>700</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>700</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <italic>false</italic>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>CrossCursor</cursorShape>
           </property>
           <property name="text">
            <string>远程摄像头未连接</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="control_system">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>200</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="title">
           <string>家电控制系统</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QPushButton" name="open_light">
             <property name="font">
              <font>
               <family>苹方-简</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string/>
             </property>
             <property name="statusTip">
              <string>一键打开家庭照明系统</string>
             </property>
             <property name="whatsThis">
              <string/>
             </property>
             <property name="text">
              <string>开灯</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>source/lamp_bulb.png</normaloff>source/lamp_bulb.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="open_fan">
             <property name="font">
              <font>
               <family>苹方-简</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="statusTip">
              <string>一键打开家庭散热系统</string>
             </property>
             <property name="text">
              <string>打开风扇</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>source/electric_fan.png</normaloff>source/electric_fan.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>300</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>300</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="title">
           <string>门禁系统</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QPushButton" name="open_camera">
             <property name="font">
              <font>
               <family>苹方-简</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>打开摄像头</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>source/camera.png</normaloff>source/camera.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="active_security">
             <property name="font">
              <font>
               <family>苹方-简</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="statusTip">
              <string>用于激活蜂鸣器装置，驱赶陌生人或警示附近的人群，提高安全性</string>
             </property>
             <property name="text">
              <string>安全响应</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>source/security_alerts.png</normaloff>source/security_alerts.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="close_alarm">
             <property name="font">
              <font>
               <family>苹方-简</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>关闭报警</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>source/close_alarm.png</normaloff>source/close_alarm.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="start_recognition">
             <property name="font">
              <font>
               <family>苹方-简</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="statusTip">
              <string>对来访者进行人脸识别，以确定其身份</string>
             </property>
             <property name="text">
              <string>开始识别</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>source/face.png</normaloff>source/face.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="title">
           <string>智能语音助手</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QPushButton" name="hold_to_talk">
             <property name="font">
              <font>
               <family>苹方-简</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>OpenHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>按住说话</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>source/voice.png</normaloff>source/voice.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
